components/Players/PlayerManager.vue

```diff
@@ -66,6 +66,106 @@
               </div>
             </v-overlay>
           </v-fade-transition>
+          <!-- Full Screen OSD -->
+          <v-fade-transition>
+            <v-overlay v-show="!isMinimized && hover" absolute>
+              <div
+                class="d-flex flex-column justify-space-between align-center player-overlay"
+              >
+                <div class="osd-top">
+                  <div class="d-flex justify-space-between align-center">
+                    <div class="d-flex">
+                      <v-btn icon @click="stopPlayback">
+                        <v-icon>mdi-close</v-icon>
+                      </v-btn>
+                      <v-btn icon @click="toggleMinimized">
+                        <v-icon> mdi-chevron-down </v-icon>
+                      </v-btn>
+                      <v-btn
+                        v-if="supportedFeatures.pictureInPicture"
+                        icon
+                        disabled
+                      >
+                        <v-icon> mdi-picture-in-picture-bottom-right </v-icon>
+                      </v-btn>
+                    </div>
+                    <p class="ma-0 text-center">{{ currentItemName }}</p>
+                    <div class="d-flex">
+                      <v-btn icon disabled>
+                        <v-icon> mdi-autorenew </v-icon>
+                      </v-btn>
+                      <v-btn v-if="supportedFeatures.airplay" icon disabled>
+                        <v-icon> mdi-apple-airplay </v-icon>
+                      </v-btn>
+                      <v-btn icon disabled>
+                        <v-icon> mdi-cast </v-icon>
+                      </v-btn>
+                    </div>
+                  </div>
+                </div>
+
+                <div class="px-4 osd-bottom">
+                  <div>
+                    <v-slider
+                      min="0"
+                      :max="runtime"
+                      validate-on-blur
+                      :step="0"
+                      :value="sliderValue"
+                      hide-details
+                      thumb-label
+                      @end="onPositionChange"
+                      @change="onPositionChange"
+                      @mousedown="onClick"
+                      @mouseup="onClick"
+                      @input="onInputChange"
+                    >
+                      <template #prepend>
+                        <span class="mt-1">
+                          {{ getRuntime(realPosition) }}
+                        </span>
+                      </template>
+                      <template #thumb-label>
+                        {{ getRuntime(sliderValue) }}
+                      </template>
+                      <template #append>
+                        <span class="mt-1">
+                          {{ getRuntime(runtime) }}
+                        </span>
+                      </template>
```

Comment:
@Camc314 Why we don't make a custom component for this? Seems useless to repeat the same logic that we use in Audio controls here.

We can name it ``PlaybackProgressSlider``.

We will also need this component for the future Full screen page for music playback, so it makes total sense imo.

Reply:
@Camc314 Use the one in Audio Controls as a template, there are some changes there that haven't been backported here (like the removal of the ``min`` prop).

Reply:
@ferferga We should probably do this in another PR, since it'll touch other components as well. I'll merge this, then we can work on unifying the common components of the various players.

Reply:
@MrTimscampi okay, will do it in the full screen PR then.